samples/Azure.Network.Management.Interface/Azure.Network.Management.Interface/Generated/Models/NetworkSecurityGroup.cs

```diff
@@ -51,11 +51,11 @@ internal NetworkSecurityGroup(string id, string name, string type, string locati
         /// <summary> A collection of security rules of the network security group. </summary>
         public IList<SecurityRule> SecurityRules { get; }
```

Comment:
Why is this not `IReadOnlyList`?

Reply:
The model itself is an intput-output model but DefaultSecurityRules, NetworkInterfaces and Subnets are readonly properties while SecurityRules is not.